components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/Extensions.kt

```diff
@@ -13,3 +14,10 @@ fun Map<String, String>.translate(): String {
     val lang = Locale.getDefault().language
     return get(lang) ?: getValue("en-US")
 }
+
+/**
+ * Get the formatted number amount for the current default locale.
+ */
+fun getFormattedAmount(amount: Int): String {
```

Comment:
nit: if we don't need this outside this package let's make it private :)

Reply:
Fixed. Made this internal